django/contrib/auth/views.py

```
@@ -104,9 +104,10 @@ def get_form_kwargs(self):
         return kwargs
 
     def form_valid(self, form):
-        """Security check complete. Log the user in."""
-        auth_login(self.request, form.get_user())
-        return HttpResponseRedirect(self.get_success_url())
+        user = form.get_user()
+        login(self.request, user)
```

Ruff (0.12.2)

Undefined name `login`

(F821)